serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -1,18 +1,16 @@
-= Kubernetes Service Discovery for Serverless Workflow Application
+= Kubernetes service discovery in {context}
 :compat-mode!:
 // Metadata:
 :description: Explain what is and how the service discovery works
 :keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
 // links
 :quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
 
+The Kubernetes service discovery allows you to have a static URI, defining a Kubernetes resource, which is used to perform HTTP requests. The Kubernetes resource defined in the URI is queried in the current Kubernetes cluster and translated in a valid URL. 
 
-The Kubernetes Service Discovery allows you to have a static URI that defines a Kubernetes resource that will be used
-to perform HTTP requests. The resource defined in the URI will then be queried in the current Kubernetes cluster and
-translated to a valid URL. The service discovery feature works during the Application startup, it scans all Quarkus
-configuration in search of the following URI pattern. Thus, you should keep in mind that, if the startup time matters to you,
-then you should consider to use a static URL instead. The following sketch demonstrates how this URI is composed:
+The Kubernetes service discovery feature works during the workflow application startup, in which this feature scans all the Quarkus configuration in search of the following URI pattern:
 
+.URI pattern in Kubernetes service discovery
```

Comment:
```suggestion
.KubeURI Pattern
```

Reply:
Please don't use `KubeURI`, this means nothing.

Reply:
So should we keep `.URI pattern in Kubernetes service discovery`?